R/bundle.R

```diff
@@ -80,7 +82,24 @@ bundleApp <- function(appName, appDir, appFiles, appPrimaryDoc, assetTypeName,
   bundlePath
 }
 
-isShinyRmd <- function(filename) {
+appHasParameters <- function(appDir, files) {
+  rmdFiles <- grep("^[^/\\\\]+\\.rmd$", files, ignore.case = TRUE, perl = TRUE,
```

Comment:
No, I don't think we need to try to figure out if a doc is nested (seems like a lot of engineering and the worst thing that could happen is a superfluous dependency on Shiny). I'm asking about the case wherein you publish several top-level docs as part of a multi-page app.